src/api/component.js

```diff
@@ -23,7 +24,6 @@ import { createNativePropertyDescriptor } from '../lifecycle/props-init';
 import { isFunction } from '../util/isType';
```

Comment:
nit: I didn't catch this before, this should follow the current convention of `is-type`, even if we decide to change this in the future to match the casing of the default export.